overlord/configstate/configstate.go

```diff
@@ -98,7 +98,7 @@ func configureHook(st *state.State, snapName string, patch map[string]interface{
 func Configure(st *state.State, snapName string, patch map[string]interface{}, flags int) *state.TaskSet {
 	// configuration for "core" is handled internally
 	if snapName == "core" {
-		return configureSnapd(st, snapName, patch, flags)
+		return configureSnapd(st, "core" patch, flags)
```

Comment:
Missing coma? Anyway, I think the name doesn't need to be passed at all.

Reply:
Indeed, silly me. Will fix